pkg/collector/corechecks/containers/containerd/adapter.go

```diff
@@ -40,6 +40,7 @@ var metricsNameMapping = map[string]string{
 	"container.net.rcvd":                       "",                                         // Not present in legacy check
 	"containerd.blkio.service_recursive_bytes": "containerd.blkio.service_recursive_bytes", // Passthrough for custom metrics extension
 	"containerd.blkio.serviced_recursive":      "containerd.blkio.serviced_recursive",      // Passthrough for custom metrics extension
+	"container.restarts":                       "containerd.restarts",
```

Comment:
We don't have metrics to existing `containerd` and `docker` checks. Only the new `container` checks gets new metrics.